### PR TITLE
[FIX] Removed the built-in len function used as condition

### PR DESCRIPTION
**Pull request by rayandas**
Using the `len` function to check if a sequence is empty is not idiomatic and can be less performant than checking the truthiness of the object.

`len` doesn't know the context in which it is called, so if computing the length means traversing the entire sequence, it must; it doesn't know that the result is just being compared to 0. Computing the boolean value can stop after it sees the first element, regardless of how long the sequence actually is.

### CODE REVIEW

File: terminusdb_client/woqlquery/woql_query.py [hunk 1]
```diff
@@ -482,7 +482,7 @@ def _find_last_subject(self, json):
     def _compile_path_pattern(self, pat):
         """Turns a textual path pattern into a JSON-LD description"""
         toks = _tokenize(pat)
-        if toks and len(toks):
+        if toks and toks:
```
Comment (Cheukting) [inline terminusdb_client/woqlquery/woql_query.py:485]: since now it's "toks and toks", we can just check "if toks"

**Review by Cheukting — CHANGES_REQUESTED** [review]
Looks good except one minor change requested.

File: terminusdb_client/woqlquery/woql_query.py (hunk 1 above)
Comment (rayandas) [reply terminusdb_client/woqlquery/woql_query.py:485]: yeah right. updating the PR.

Comment (rayandas) [reply terminusdb_client/woqlquery/woql_query.py]: @Cheukting updated the PR

**Review by Cheukting — APPROVED** [review]
Looks good, thanks!

**Pull request merged**